pkg/awsclient/mock/client_generated.go

```diff
@@ -783,3 +784,60 @@ func (mr *MockClientMockRecorder) WaitUntilVpcPeeringConnectionExists(arg0 inter
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "WaitUntilVpcPeeringConnectionExists", reflect.TypeOf((*MockClient)(nil).WaitUntilVpcPeeringConnectionExists), arg0)
 }
+
+// MockIPaginator is a mock of IPaginator interface.
```

Comment:
Ugh. This showed up when we regenerated. It looks like it was somehow missed in the AWS SDK v2 migration... but for some reason `verify` didn't pick it up. I should rip it out of here and put it in a separate PR.

/hold